packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -628,7 +590,8 @@ const NavBar = <AdditionalEnvironmentProperties extends {}>(
     handleToggleItem,
     handleToggleMenu,
     shouldCloseMenuFly,
-    allApplicationNavbarMenu,
+    allInternalApplicationsNavbarMenu,
+    allCustomApplicationsNavbarMenu,
```

Comment:
I opted to split up the internal from external apps as there is a divider between the two groups.